zpay32/invoice.go

```diff
@@ -271,37 +274,41 @@ func Decode(invoice string) (*Invoice, error) {
 	if err != nil {
 		return nil, err
 	}
-
-	// We expect the human-readable part to at least have ln + two chars
-	// encoding the network.
-	if len(hrp) < 4 {
-		return nil, fmt.Errorf("hrp too short")
-	}
-
 	// First two characters of HRP should be "ln".
 	if hrp[:2] != "ln" {
 		return nil, fmt.Errorf("prefix should be \"ln\"")
 	}
 
 	// The next characters should be a valid prefix for a segwit BIP173
 	// address. This will also determine which network this invoice is
-	// meant for.
+	// meant for. RegressionNet and TestNet share the same prefix.
```

Comment:
Nit: No need to mention this. They share the same prefix, but have different parameters.